meta/contributor-handbook.md

```diff
@@ -26,37 +26,37 @@ To achieve a given maturity level a pattern has to satisfy the requirements for
 Patterns (or other documents) of level 1 are stored in the directory `/patterns/1-initial`.
 
 - Validation requirements:
-	- N/A
-	
+  - N/A
+ 
 - Content requirements:
-	- The document is readible & comprehensible for other parties (not just for the author(s))
-	- The author(s) contribute the contents according to the [license](../LICENSE.txt) & are allowed to do so
-	- Thoughts and contents by third parties are quoted / referenced explicitly
+  - The document is readible & comprehensible for other parties (not just for the author(s))
+  - The author(s) contribute the contents according to the [license](../LICENSE.txt) & are allowed to do so
+  - Thoughts and contents by third parties are quoted / referenced explicitly
 
-	
+ 
 ### Requirements: Level 2 - Structured
 
 Patterns of level 2 are stored in the directory `/patterns/2-structured`.
 
 - Validation requirements:
-	- Is validated by at least one known instance 
-		- Alternatively: key elements of the pattern have been validated in separate contexts and, in consequence, it is justified to believe the full solution will function
-	
+  - Is validated by at least one known instance 
+  - Alternatively: key elements of the pattern have been validated in separate contexts and, in consequence, it is justified to believe the full solution will function

```

Comment:
I checked the visual diff on this file, and the bullet point starting with "Alternatively" was actually meant to be one level lower. 

Therefore we should change it to:

```suggestion
  - Is validated by at least one known instance 
    - Alternatively: key elements of the pattern have been validated in separate contexts and, in consequence, it is justified to believe the full solution will function
```